scripts/apply_def_template.py

```diff
@@ -17,88 +17,89 @@
 import re
 import click
 import shutil
-import argparse
 import subprocess
 
+
 @click.command()
 @click.option("-t", "--def-template", "templateDEF", required=True, help="Template DEF")
 @click.argument("userDEF")
 def cli(templateDEF, userDEF):
     scriptsDir = os.path.dirname(__file__)
 
     def remove_power_pins(DEF):
-        templateDEFOpener = open(DEF,"r")
-        if templateDEFOpener.mode == 'r':
-            templateDEFSections =templateDEFOpener.read().split("PINS")
+        templateDEFOpener = open(DEF, "r")
+        if templateDEFOpener.mode == "r":
+            templateDEFSections = templateDEFOpener.read().split("PINS")
         templateDEFOpener.close()
         PINS = templateDEFSections[1].split("- ")
-        OUT_PINS=[" ;"]
+        OUT_PINS = [" ;"]
         cnt = 0
         for pin in PINS[1:]:
             if pin.find("USE GROUND") + pin.find("USE POWER") == -2:
-                cnt+=1 
+                cnt += 1
                 OUT_PINS.append(pin)
-        OUT_PINS[0] = " "+str(cnt) + OUT_PINS[0] + PINS[0].split(";")[1]
+        OUT_PINS[0] = " " + str(cnt) + OUT_PINS[0] + PINS[0].split(";")[1]
         OUT_PINS[-1] = OUT_PINS[-1].replace("END ", "")
         OUT_PINS[-1] = OUT_PINS[-1] + "END "
         templateDEFSections[1] = "- ".join(OUT_PINS)
-        templateDEFOpener = open(DEF,"w")
+        templateDEFOpener = open(DEF, "w")
         templateDEFOpener.write("PINS".join(templateDEFSections))
         templateDEFOpener.close()
-        
+
     newTemplateDEF = f"{userDEF}.template.tmp"
     shutil.copy(templateDEF, newTemplateDEF)
     templateDEF = newTemplateDEF
 
     templateDEF = f"{userDEF}.template.tmp"
     remove_power_pins(templateDEF)
 
-
-    subprocess.check_output([
-        "openroad",
-        "-python",
-        f"{scriptsDir}/defutil.py",
-        "--output", userDEF,
-        "--input-lef", "/dev/null",
-        userDEF, templateDEF
-    ], stderr=subprocess.PIPE)   
-
-    #read template Def
-    templateDEFOpener = open(templateDEF,"r")
-    if templateDEFOpener.mode == 'r':
-        templateDEFContent =templateDEFOpener.read()
+    subprocess.check_output(
+        [
+            "openroad",
+            "-python",
+            "move_pins" f"{scriptsDir}/odbpy/defutil.py",
```

Comment:
I am not sure but something looks odd here

Reply:
something IS odd here, there should be a comma